cinder/volume/targets/rdma.py

```diff
@@ -21,13 +21,11 @@ class RDMATarget(driver.Target):
     """Target object for block storage devices with RDMA transport.
     """
 
-    def __init__(self, *args, **kwargs):
-        super(RDMATarget, self).__init__(*args, **kwargs)
-        self.protocol = 'RDMA'
+    PROTOCOL = 'RDMA'
```

Comment:
Please, use lower-case protocol naming here (e.g. 'rdma'). It would keep compatibility with current Nova and Cinder things for PoC